xapian-core/cluster/euclidian_sim.cc

```diff
@@ -0,0 +1,75 @@
+#include <config.h>
```

Comment:
No, not a header file: there's no info and copyright header in _this_ file, such as there is in our other source files.